extensions/scheduler/runtime/pom.xml

```diff
@@ -45,6 +45,12 @@
             <artifactId>quarkus-vertx-http</artifactId>
             <optional>true</optional>
         </dependency>
+        <!-- TEST dependencies -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5-internal</artifactId>
```

Comment:
We don't need the quarkus junit extension here, just a plain junit dependency would be enough...

Reply:
Sure